### PR TITLE
adding GlobalizationExtensions APIs

### PR DESCRIPTION
**Pull request by mairaw**
@krwq, @tarekgh can one of you review the docs for the GlobalizationExtensions class and its APIs?

[Internal review URL](https://review.docs.microsoft.com/en-us/dotnet/api/system.globalization.globalizationextensions?view=netcore-2.0&branch=pr-en-us-3263)

### CODE REVIEW

File: xml/System.Globalization/GlobalizationExtensions.xml [hunk 1]
```diff
@@ -53,11 +53,15 @@
         <Parameter Name="options" Type="System.Globalization.CompareOptions" />
       </Parameters>
       <Docs>
-        <param name="compareInfo">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="compareInfo">An object that supports culture-sensitive string comparison.</param>
+        <param name="options">A value that defines how strings should be compared. <c>options</c> is either the enumeration value  <see cref="F:System.Globalization.CompareOptions.Ordinal" /> or a bitwise combination of one or more of the following values: <see cref="F:System.Globalization.CompareOptions.IgnoreCase" />, <see cref="F:System.Globalization.CompareOptions.IgnoreSymbols" />, <see cref="F:System.Globalization.CompareOptions.IgnoreNonSpace" />, <see cref="F:System.Globalization.CompareOptions.IgnoreWidth" />, <see cref="F:System.Globalization.CompareOptions.IgnoreKanaType" />, and <see cref="F:System.Globalization.CompareOptions.StringSort" />.</param>
```
Comment (tarekgh) [inline xml/System.Globalization/GlobalizationExtensions.xml]: >s either the enumeration value  <see cref="F:System.Globalization.CompareOptions.Ordinal" /> [](start = 99, length = 92)

The options can also be CompareOptions.OrdinalIgnoreCase

**Comment by tarekgh** [comment]
Left a small comment. otherwise LGTM.

**Review by BillWagner — APPROVED** [review]
This LGTM.

Once you have the product team approval, :shipit:

**Comment by mairaw** [comment]
Thanks @tarekgh. I've addressed your comment.

**Review by tarekgh — APPROVED** [review]

**Comment by rpetrusha** [comment]
The build completed but its status isn't reported here. Closing and reopening to begin new build.

**Pull request merged**